### PR TITLE
Add global statistic

### PR DESCRIPTION
**Pull request by Jimmba**

### CODE REVIEW

File: src/js/main page/globalStatisticData.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    for (let i = 0; i < datesArray.length; i+= 1) {
+        data.push(getDatesWithValues(datesArray[i], valuesArray[i]));
+    }
+    return data;
+}
+
+const getData = (data) => {
+    const dates = Object.keys(data).map(el => new Date(el).getTime());
+    let values = Object.values(data);
+    console.log(values);
```
Comment (mill9r) [inline src/js/main page/globalStatisticData.js:49]: Do we need console.log?

File: src/js/main page/globalStatisticData.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    let accumulator = 0;
+    for (let i = 0; i < values.length; i += 1) {
```
Comment (mill9r) [inline src/js/main page/globalStatisticData.js:51]: from my point of view for (let i = 0; i < values.length; i++ ) is more convenient

File: src/js/main page/globalStatisticData.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        accumulator += values[i];
+        values[i] = accumulator;
+    }
+    console.log(values);
```
Comment (mill9r) [inline src/js/main page/globalStatisticData.js:55]: the same

File: src/js/main page/globalStatisticData.js (hunk 2 above)
Comment (Jimmba) [reply src/js/main page/globalStatisticData.js:51]: image
I can't use i++ by Lint rules.

File: src/js/main page/globalStatisticData.js (hunk 1 above)
Comment (Jimmba) [reply src/js/main page/globalStatisticData.js:49]: fixed

File: src/js/main page/globalStatisticData.js (hunk 3 above)
Comment (Jimmba) [reply src/js/main page/globalStatisticData.js]: fixed

**Pull request merged**